charts/ingress-configuration/values.yaml

```diff
@@ -0,0 +1,27 @@
+releaseServiceOverride: ArgoCD
+
+defaultReplicas: 2
+
+replicasOverride:
+  hub: 3
+
+defaultLoggingRequestMaxlength:
+  XForwardedFor: 256
+  XForwardedProto: 128
+  XForwardedPort: 6
+  XForwardedhost: 256
+  Forwarded: 256
+
+loggingRequestMaxlengthOverride:
```

Comment:
@skassovic , hub configuration shall be in conf repo in rendering values. suggest to change it.